src/CommunityToolkit.Maui.Core/Views/Popup/MauiPopup.tizen.cs

```diff
@@ -0,0 +1,145 @@
+﻿using Microsoft.Maui.Platform;
+using Microsoft.Maui.Primitives;
+using Tizen.NUI;
+using Tizen.UIExtensions.NUI;
+using NHorizontalAlignment = Tizen.NUI.HorizontalAlignment;
+using NVerticalAlignment = Tizen.NUI.VerticalAlignment;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+/// <summary>
+/// The native implementation of Popup control.
+/// </summary>
+public class MauiPopup : Popup
+{
+	readonly IMauiContext mauiContext;
+
+	/// <summary>
+	/// Constructor of <see cref="MauiPopup"/>.
+	/// </summary>
+	/// <param name="mauiContext">An instance of <see cref="IMauiContext"/>.</param>
+	/// <exception cref="ArgumentNullException">If <paramref name="mauiContext"/> is null an exception will be thrown. </exception>
+	public MauiPopup(IMauiContext mauiContext)
+	{
+		this.mauiContext = mauiContext ?? throw new ArgumentNullException(nameof(mauiContext));
+		OutsideClicked += OnOutsideClicked;
+	}
+
+	/// <inheritdoc/>
+	protected override void Dispose(bool isDisposing)
+	{
+		if (isDisposing)
+		{
+			OutsideClicked -= OnOutsideClicked;
+		}
+
+		base.Dispose(isDisposing);
+	}
+
+	void OnOutsideClicked(object? sender, EventArgs e)
+	{
+		if (VirtualView is null || VirtualView.Handler is null)
+		{
+			return;
+		}
+
+		if (VirtualView.CanBeDismissedByTappingOutsideOfPopup)
+		{
+			Close();
+			VirtualView.Handler.Invoke(nameof(IPopup.OnDismissedByTappingOutsideOfPopup));
+		}
+	}
+
+	/// <summary>
+	/// An instance of the <see cref="IPopup"/>.
+	/// </summary>
+	public IPopup? VirtualView { get; private set; }
```

Comment:
can we move this to the top of the class?

Reply:
Sure, I reordered layout of class